modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
+
+To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class in the `openshift-cluster-csi-drivers` namespace by default.
```

Comment:
@bobfuru 
1. In OCP4.6 and OCP4.7, manila csi driver operator is installed in "openshift-cluster-csi-drivers" namespace, but manila csi driver is installed in "openshift-manila-csi-driver". @jsafrane By this chance, could you share us is there any plan to move manila to "openshift-cluster-csi-drivers" or we will keep it.
2.  Normally, we don't say "storage class" as namespace wide resource, better to separate them.

Reply:
To simplify, I have more generically stated this and referred the reader to the Driver Operator-specific documentation:
`To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class by default. For more details about the default namespace of the Operator and driver, see the documentation for the specific CSI Driver Operator.`